### ISSUE TITLE
Default theme color used by antd does not meet sufficient color contrast ratio

### ISSUE THREAD

**Issue by ekawatani** [ant-design/ant-design#28760]
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://ant.design/components/button/](https://ant.design/components/button/)

### Steps to reproduce
The color contrast ratio between the background color (Primary: `#1788f1`, Danger: `#f1494b`) and the text color (`#ffffff`) is too low.

**Method 1**
This can be checked directly in Google Chrome.
1. Open the Chrome developer tool by pressing F12.
2. Click on "Select an element on the page to inspect it" in the top left corner.
3. Hover the cursor on the text in the button.

Then, Chrome shows a pop-up message that includes color contrast information.
![image](https://user-images.githubusercontent.com/61330990/103885767-cc92c800-50e0-11eb-8d39-0af117f0f4f4.png)

**Method 2**
You can also check this using tools such as Accessibility Insights tools for Web (available via Chrome Web Store).

### What is expected?
The color contrast ratio of background and foreground color should be at least 4.5.

### What is actually happening?
The color contrast ratio is around 3.
I suggest changing the default colors to meet a higher color contrast. It should not be done as part of a special "accessibility" mode and it should be built into the product by default.

| Environment | Info |
|---|---|
| antd | 4.10.0 |
| React | latest |
| System | All systems |
| Browser | All browsers |

---
This is important for accessibility purposes. Some visually impaired people will not be able to or have very hard time using products created in antd with the default theme. Operating systems such as Windows have high contrast mode, but using this ruins the user experience for most people. Usually, all we need is sufficient color contrast ratios.

**Comment by monolithed** [comment]
The problem is actual for `hover` state as well. Why not support customizing Less variables to change one?.

**Comment by stefanonardo** [comment]
Any updated on this?

**Comment by afc163** [comment]
- Same issue: https://github.com/ant-design/ant-design/issues/49543

**Comment by hugobarragon** [comment]
@afc163 hello, we are on V5 now and this is the next major version and its not meeting the minium AA 4.5:1 contrast, at least the minium, any updates or how to fix this in the entire theme?

**Comment by leo29plns** [comment]
Please

**Comment by malparty** [comment]
Not sure if related, but it's a bit sad that on the Value pages the contrasts are that low... https://ant.design/docs/spec/values
<img width="1691" height="562" alt="Image" src="https://github.com/user-attachments/assets/0d776693-1bf5-419e-acb7-e30c74329972" />

**Comment by hugobarragon** [comment]
@afc163 
what abound including this on ANTV V6 ? 
https://github.com/ant-design/ant-design/issues/52115

**Comment by stefanonardo** [comment]
Or maybe just add a disclaimer to let the users know that Antd is not supposed to meet all the accessibility requirements.
